apps/explorer-e2e/src/integration/market.cy.js

```diff
@@ -88,14 +88,6 @@ context('Market page', { tags: '@regression' }, function () {
       cy.validate_element_from_table('Target Stake', '0.00 fUSDC');
       cy.validate_element_from_table('Supplied Stake', '0.00 fUSDC');
       cy.validate_element_from_table('Market Value Proxy', '0.00 fUSDC');
-      // Liquidity price range
```

Comment:
Tests are only removed, but new panel where added to market info. This should be covered by tests.

Reply:
agree. updated.